test/controllers/testOnly/FeatureSwitchControllerSpec.scala

```diff
@@ -38,7 +38,7 @@ class FeatureSwitchControllerSpec extends SpecBase with FeatureSwitching {
     reset(mockConfig)
     reset(mockServicesConfig)
     val controller = new FeatureSwitchController(mcc)(config)
-    FeatureSwitch.listOfAllFeatureSwitches.foreach(sys.props -= _.name)
```

Comment:
My bad, you can keep this `FeatureSwitch.listOfAllFeatureSwitches.foreach(sys.props -= _.name)` but on the line below add: `sys.props -= TIME_MACHINE_NOW`